src/Connections/GithubRepo.ts

```diff
@@ -678,7 +678,9 @@ export class GitHubRepoConnection extends CommandConnection<GitHubRepoConnection
                 throw new CommandError('No issues found', 'There are no issues on this repository');
             }
             issueNumber = topIssue.number;
-        }
+        } else {
+            issueNumber = parseInt(number, 10);
```

Comment:
If `number` is a non-numberable string, this will set `issue_number` to `NaN`. Is that ok?

Reply:
Yeah, we can let GitHub explode as we did before.